src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+// import { store } from '../../app/store';
+import { actions as filterActions } from '../../features/filter';
+import { Status } from '../../types/Status';
+import { useAppSelector } from '../../app/hooks';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useDispatch();
+  const filter = useAppSelector(state => state.filter);
```

Comment:
```suggestion
  const query = useAppSelector(state => state.filter.query);
```
makes sense to get only query if you don't need `status` here